php/Terminus/Commands/SiteCommand.php

```diff
@@ -2142,25 +2130,19 @@ public function upstreamUpdates($args, $assoc_args) {
             isset($assoc_args['accept-upstream'])
             && $assoc_args['accept-upstream']
           );
-          $message  = 'Are you sure you want to apply the ';
-          $message .= 'upstream updates to %s-dev';
+          $message        = 'Are you sure you want to apply the ';
+          $message       .= 'upstream updates to %s-dev';
           $this->input()->confirm(
             array(
               'message' => $message,
               'context' => array($site->get('name'), $env),
             )
           );
-          $workflow = $site->applyUpstreamUpdates(
-            $env,
-            $updatedb,
-            $acceptupstream
-          );
+          $workflow = $site->applyUpstreamUpdates($env, $updatedb, $acceptupstream);
```

Comment:
This does not appear to work as it triggers a site workflow that should be an environment workflow:

https://github.com/pantheon-systems/terminus/pull/1223/files#diff-7d2978652d899c64c9abd2f4cc49f103R753